themes/blue-cheese-italics-color-theme.json

```diff
@@ -0,0 +1,1904 @@
+{
+  "name": "Blue Cheese",
```

Comment:
```suggestion
  "name": "Blue Cheese Italics",
```

Reply:
All good - I'll fix these things and resubmit